guide/how-it-works/human-readable-addresses.md

```diff
@@ -0,0 +1,285 @@
+   height = 334
+%}
+
+On the one hand, the input field cannot be optimized just for entering a single format. On the other hand, it is highly unlikely that users type in the more complex address types. We can find a balance of allowing diverse input, while also optimizing for entering address types that are designed for manual input.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/human-readable-addresses/input.png"
+   retina = "/assets/images/guide/how-it-works/human-readable-addresses/input@2x.png"
+   mobile = "/assets/images/guide/how-it-works/human-readable-addresses/input-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/human-readable-addresses/input-mobile@2x.png"
+   alt-text = "Several states of an address input field showing how a user might enter bitcoin addresses"
+   width = 800
+   height = 452
+%}
+
+#### Keeping users informed
+
+The payment information associated with addresses in contacts may change or even become invalid. Wallets should ensure to inform the user of updates to review or act on.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/human-readable-addresses/update.png"
+   retina = "/assets/images/guide/how-it-works/human-readable-addresses/update@2x.png"
+   mobile = "/assets/images/guide/how-it-works/human-readable-addresses/update-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/human-readable-addresses/update-mobile@2x.png"
+   alt-text = "Two modals notifying the user of changed and disabled bitcoin addresses"
+   width = 800
+   height = 277
+%}
+
+## Lightning Address
+
+Compared to DNS Payment Instructions, [Lightning Addresses](http://lightningaddress.com) are focused on the lightning network only, bringing another intermediary into play, allowing for much more dynamic functionality.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/human-readable-addresses/lightning-address-mapping.png"
+   retina = "/assets/images/guide/how-it-works/human-readable-addresses/lightning-address-mapping@2x.png"
+   mobile = "/assets/images/guide/how-it-works/human-readable-addresses/lightning-address-mapping-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/human-readable-addresses/lightning-address-mapping-mobile@2x.png"
+   alt-text = "Diagram showing how a lightning address is transformed to a URL"
+   width = 800
+   height = 157
+%}
+
+### How it works
+
+Lightning addresses look like traditional emails. Under the hood, a wallet that uses a lightning address reformats it to a URL. It then looks for information about a [LNURL](https://github.com/lnurl/luds) web-server at that location (more on that [here](https://github.com/lnurl/luds/blob/luds/16.md)). That server handles the communication with the lightning node and passes messages back and forth. This allows for complex interactions (like authentication and withdrawals), but also adds technical overhead and requires trusting the web server.
```

Comment:
```suggestion
Lightning addresses look like traditional emails. Under the hood, a wallet that uses a lightning address reformats it to a URL. It then looks for information about a [LNURL](https://github.com/lnurl/luds) web-server at that location (more on that [here](https://github.com/lnurl/luds/blob/luds/16.md)). That server handles the communication with the lightning node and passes messages back and forth. This allows for complex interactions (like authentication and withdrawals), but also requires trusting the web server.
```

While I absolutely love the DNS payment instructions, to be completely fair, it also requires technical overhead to implement. So I don't think it's accurate or fair to characterize Lightning address as "this adds technical overhead".